### PR TITLE
Enable typechecks for torch.nn.modules.[batchnorm|instancenorm]

### PR DESCRIPTION
**Pull request by walterddr**

### CODE REVIEW

**Comment by walterddr** [comment]
close for now until fixing the issue of register_parameters/register_buffer when jit uses type syntax suger
